lib/fake_braintree/transaction.rb

```diff
@@ -12,7 +12,8 @@ def create
         "id" => id,
         "amount" => data["amount"],
         "status" => status,
-        "type" => "sale"
+        "type" => "sale",
+        "created_at" => Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.